packages/commonwealth/client/scripts/views/pages/user_dashboard/user_dashboard_row_bottom.tsx

```diff
@@ -23,6 +24,22 @@ type UserDashboardRowBottomProps = {
 
 export const UserDashboardRowBottom = (props: UserDashboardRowBottomProps) => {
   const { threadId, commentCount, commentId, chainId, commenters } = props;
+  const [update, setUpdate] = useState<boolean>(false);
+
+  const setSubscription = async (
+    threadId: string,
+    bothActive: boolean,
+    commentSubscription: NotificationSubscription,
+    reactionSubscription: NotificationSubscription
+  ) => {
+    await subscribeToThread(
+      threadId,
+      bothActive,
+      commentSubscription,
+      reactionSubscription
+    );
+    setUpdate(!update); // just to trigger a new render
+  };
```

Comment:
I was tempted to turn `helpers.ts` into a custom hook so this state update could be included there, but that is a bit of a heavy pattern update. I will suggest this in future PRs, but I'll keep the diff light for now.